xtask/src/main.rs.liquid

```diff
@@ -49,12 +49,14 @@ fn check(subcommand: Option<&str>) {
 
     match subcommand {
         Some("commits") => check_commits(&mut ctx),
+        Some("doc") => check_doc(&mut ctx),
```

Comment:
**issue:** The CI configuration should be updated as well to add a new job.

Reply:
I've never used github actions, I think it'd be way quicker to let you do that if you don't mind.